streams/writable-streams/aborting.js

```diff
@@ -157,16 +163,15 @@ promise_test(t => {
 
   const writer = ws.getWriter();
 
-  writer.write().catch(() => {});
+  writer.write(1).catch(() => {});
```

Comment:
We treat `write()` as `write(undefined)`. `undefined` is a valid chunk.

I think you might be right that this is unspec'd. The "in something close to the syntax of [ECMASCRIPT]," sections are non-normative, and the steps don't say what happens if arguments are omitted. @domenic is this covered somewhere, or do we need to add an explicit normative note?